src/setup/platformSetup/index.native.ts

```diff
@@ -12,14 +10,5 @@ export default function () {
         crashlytics().setCrashlyticsCollectionEnabled(false);
     }
 
-    /*
-     * Register callbacks for push notifications.
-     * When the app is completely closed, this code will be executed by a headless JS process thanks to magic in the UrbanAirship RN library.
-     * However, the main App component will not be mounted in this headless context, so we must register these callbacks outside of any React lifecycle.
-     * Otherwise, they will not be executed when the app is completely closed, and the push notification won't update the app data.
-     */
-    PushNotification.init();
-    subscribeToReportCommentPushNotifications();
```

Comment:
This subscription was a duplicate and has been unnecessary for a while

Reply:
We've been handling the subscription here so it's a dupe:
https://github.com/Expensify/App/blob/56424115c791d15bc44273c32a8c38e96f02165c/src/libs/Notification/PushNotification/subscribePushNotification/index.ts#L111-L135

The callbacks should work there, but I haven't been able to build Android to test it. Can you build Android to test?

Reply:
Nvm I fixed the build [here](https://expensify.slack.com/archives/C01GTK53T8Q/p1715348358384259). Background updates are still working well on Android:

![Screenshot 2024-05-10 at 10 40 26 AM](https://github.com/Expensify/App/assets/6833644/d4297a2a-58e9-4bdd-a021-08f2b5b88a21)